examples/step-46/step-46.cc

```diff
@@ -224,18 +224,12 @@ namespace Step46
     : stokes_degree(stokes_degree)
     , elasticity_degree(elasticity_degree)
     , triangulation(Triangulation<dim>::maximum_smoothing)
-    , stokes_fe(FE_Q<dim>(stokes_degree + 1),
-                dim,
-                FE_Q<dim>(stokes_degree),
-                1,
-                FE_Nothing<dim>(),
-                dim)
-    , elasticity_fe(FE_Nothing<dim>(),
-                    dim,
-                    FE_Nothing<dim>(),
-                    1,
-                    FE_Q<dim>(elasticity_degree),
-                    dim)
+    , stokes_fe(FE_Q<dim>(stokes_degree + 1) ^ dim, // for the fluid velocity
+                FE_Q<dim>(stokes_degree),           // for the fluid pressure
+                FE_Nothing<dim>() ^ dim)          // for the solid displacement
+    , elasticity_fe(FE_Nothing<dim>() ^ dim,      // for the fluid velocity
+                    FE_Nothing<dim>(),            // for the fluid pressure
+                    FE_Q<dim>(elasticity_degree)) // for the solid displacement
```

Comment:
It looks like the last four comments on the RHS could do with a little nudge so that they all line up.

Reply:
That would require getting rid of a certain tool that does this automatically in a way so that the last comment in the first group of 3 does not align with the previous two lines :-(

Reply:
I guess I walked into that one. For some reason (and I don't quite know why) I thought that CT let one manage the indentation of end-line comments one's self.

Reply:
:-)

Reply:
You can always use `clang-format off` in places where you want to be explicit (and probably filter it out when running doxygen).